### ISSUE TITLE
Is there a version of pythoch3d that supports cuda10.1 and python 3.7~3.9

### ISSUE THREAD

**Issue by gjgjgjfff** [facebookresearch/pytorch3d#1442]
## ❓ Questions on how to use PyTorch3D

NOTE: Please look at the existing list of Issues tagged with the label ['question`](https://github.com/facebookresearch/pytorch3d/issues?q=label%3Aquestion) or ['how-to`](https://github.com/facebookresearch/pytorch3d/issues?q=label%3A%22how+to%22). **Only open a new issue if you cannot find an answer there**.

Also note the following:

1. If you encountered any errors or unexpected issues while using PyTorch3D and need help resolving them,
   please use the "Bugs / Unexpected behaviors" issue template.

2. We do not answer general machine learning / computer vision questions that are not specific to
   PyTorch3D, such as how a model works or what algorithm/methods can be
   used to achieve X.

**Comment by bottler** [comment]
No. The latest version of PyTorch3D doesn't have builds for cuda10.1, just like PyTorch itself doesn't. You can build it yourself thought.

**Comment by MontaEllis** [comment]
But how to build? Use pip install -e pytorch3d?

**Comment by bottler** [comment]
Follow INSTALL.md. Basically, you can install the dependencies, clone the repo, and use `pip install -e .`.

**Comment by MontaEllis** [comment]
Thanks. But I wonder if pytorch3d supports python3.6 now?

**Comment by bottler** [comment]
No. Python 3.6 and 3.7 are dead and you'll find newer Python features used in the current code. If you had to I think you could work around it.

**Comment by MontaEllis** [comment]
Can I use previous vision of Pytorch3D in python3.6?

**Comment by bottler** [comment]
Possibly, but I can't help you on that. We don't support it any more. You could go through version tags (beginning `v0.`) in this repo and look at INSTALL.md. You can also look through the pytorch3d anaconda channel for python3.6 binaries.
